src/assets/thirdPartyServices/mozi/gene_annotation_service/tables/index.js

```diff
@@ -180,13 +180,17 @@ const ResultTables = props => {
                             </TableCell>
                             <TableCell>
                               <Typography variant="body1">
-                                <a
-                                  href={`https://www.ebi.ac.uk/chebi/searchId.do?chebiId=${values[2].slice(
-                                    values[2].indexOf(":") + 1
-                                  )}`}
-                                >
-                                  {values[2]}
-                                </a>
+                                {values[2] ? (
```

Comment:
Try using Gaurd Clause, instead of Ternary operators. The code will be cleaner. [I hope this reference link will be helpful](https://medium.com/@ichbinvivek/using-guard-clauses-for-cleaner-react-code-77360e648659)